evaluator/evaluate.go

```diff
@@ -164,25 +159,45 @@ func updateSystemPackages(tx *gorm.DB, system *models.SystemPlatform,
 	}
 
 	toStore := make([]models.SystemPackage, 0, len(updates))
-	for name, updateData := range updates {
+	for nevraStr, updateData := range updates {
 		var pkgUpdates []models.PackageUpdate
-
-		for _, upData := range updateData.UpdateData.AvailableUpdates {
+		// Parse each NEVRA in the input
+		nevra, err := utils.ParseNevra(nevraStr)
+		if err != nil {
+			utils.Log("nevra", nevraStr).Warn("Invalid nevra")
```

Comment:
Would be good to count these invalid states, I know we can do it in another task.